src/content/docs/devtools/Docker/docker-swarm-case-study.md

```diff
@@ -0,0 +1,206 @@
+| Attribute  | Details                                                      |
+|------------|--------------------------------------------------------------|
+| Title      | Docker swarm case study                                            |
+| Author     | Malene Ivy Olsen                                   |
+| Tags       | docker, container, containerization, swarm, container orchestration, kubernetes | <br/>
+<br/>
+
+# Docker Swarm - container orchestration tool
+[Acronyms used in the text](#acronyms-used)
+## Introduction
+This case study aims to compare Docker and Kubernetes, the two key technologies in containerization and orchestration. I'll begin by introducing Docker, an essential tool for containerization, to set the groundwork for understanding container orchestration tools. <br/>
+Then I'll dive into comparison of Docker Swarm and Kubernetes to help students/beginners understand these important tools in modern software development. 
```

Comment:
```suggestion
Then we will dive into comparison of Docker Swarm and Kubernetes to help students/beginners understand these important tools in modern software development. 
```

We do not use the first person in technical literature. If you need to refer to yourself you can as `The Author`, for example:

> In the Author's opinion...